src/go/k8s/hack/get-redpanda-info.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -x
+
+script_namespace="${1:-$NAMESPACE}"
+
+if [[ -z $script_namespace ]]; then
+  echo "Error: no NAMESPACE env defined nor script first argument"
+  exit 1
+fi
+
+ARTIFACTS_PATH=../../_e2e_artifacts/$TEST_NAME
+mkdir -p $ARTIFACTS_PATH
+kubectl -n $script_namespace get pods -o yaml >$ARTIFACTS_PATH/pods.yaml
+
+for cl in $(kubectl -n $script_namespace get cluster --output=jsonpath='{.items..metadata.name}'); do
+  replication_factor=$(kubectl -n $script_namespace get cluster $cl --output=jsonpath='{.spec.replicas}')
+
+  tls_enabled=$(kubectl -n $script_namespace get cluster $cl --output=jsonpath='{.spec.configuration.adminApi[0].tls.enabled}')
```

Comment:
Do we assume that the first listener is the local one?

Reply:
Yes, this is an assumption. I can create function that could iterate through all `adminApi` listeners and get the port number from the one that doesn't have external option enabled.

Reply:
I think we can accept it